Gopkg.toml

```diff
@@ -88,6 +88,11 @@ ignored = [
   name = "honnef.co/go/tools"
   branch = "master"
 
+# https://github.com/gogo/protobuf/commit/620da83
+[[constraint]]
+name = "github.com/gogo/protobuf"
+branch = "master"
```

Comment:
Do you not want to constrain to a `sha`? You'll force the next person trying to bump something unrelated to think whether updating gogoproto is a good idea.

Reply:
We don't constraint anything else to a SHA, save for the collation tables, which must really never ever change. Since `dep ensure -update` allows you to specify exactly the packages you want to bump, I don't think it's a problem the way it was with glide.